src/sentry/static/sentry/app/icons/icon-checkmark-sm.svg

```diff
@@ -0,0 +1,11 @@
+<svg viewBox="0 0 9 9">
+  <g
+    stroke="currentColor"
+    strokeWidth="1"
+    fill="none"
+    strokeLinecap="round"
+    strokeLinejoin="round"
```

Comment:
I had to manually camelCase all of these attributes to js-ify them for React. In the [SVG spec](https://www.w3.org/TR/SVG11/painting.html#StrokeLinejoinProperty), they're hyphenated e.g. `stroke-linejoin="round"`. Is this something we should revert back to or does it matter?

Reply:
yes! good call!!